NRFX-3926: Add links to pmic documentation (#43)
NordicSemiconductor/npmx-zephyr: doc/index.rst

      export ZEPHYR_BASE=<path_to_zephyr_directory>

> In order to run a custom application please refer to the `documentation`_ to learn how to properly configure the required setup.

Reviewer: ```suggestion
Configure a required setup and run a custom application based on the `nPM1300 documentation`_.
```
